install/kubernetes/helm/istio/charts/security/values.yaml

```diff
@@ -7,6 +7,7 @@ selfSigned: true # indicate if self-signed CA is used.
 createMeshPolicy: true
 nodeSelector: {}
 tolerations: []
+healthChecking: false
```

Comment:
This should have a comment directly above it describing it. Otherwise users don't know how to use it or what it does. It gets updated at https://istio.io/docs/reference/config/installation-options/

Reply:
Done.